dashboard/app/models/user_permission.rb

```diff
@@ -40,4 +42,17 @@ class UserPermission < ActiveRecord::Base
   ].freeze
 
   validates_inclusion_of :permission, in: VALID_PERMISSIONS
+
+  after_save :log_permission_save
+  before_destroy :log_permission_delete
+
+  def log_permission_save
+    HipChat.log "Updating UserPermission: user ID: #{self.user.id}, email: #{self.user.email}, permission: #{self.permission}",
```

Comment:
Is this going to log just on changes or on every create for new users?

What slack channel does this log to?